lib/noble.js

```diff
@@ -52,6 +52,10 @@ function Noble(bindings) {
 
 util.inherits(Noble, events.EventEmitter);
 
+Noble.prototype.stop = function() {
+  this._bindings._hci.stop();
```

Comment:
It would be nice for the bindings to have their own `.stop()` function instead of calling the one on `_hci`.

Then someone can fill in the mac version and other bindings `stop()` function. For mac we'll need for this to be implemented in a dependency of noble: https://github.com/sandeepmistry/node-xpc-connection/issues/16